users/views.py

```diff
@@ -123,8 +125,9 @@ def _generate_verification_token(self):
     
     def _send_verification_email(self, request, user, token):
         """Send verification email to the user."""
-        verification_relative_url = reverse('verify-email', kwargs={'user_id': user.user_id, 'token': token})
-        verification_url = f"{request.scheme}://{request.get_host()}{verification_relative_url}"
+        full_path = reverse("verify-email", kwargs={"user_id": user.user_id, "token": token})
+        verification_relative_url = full_path.replace("/api/v1/", "", 1)
```

Comment:
Direct string replacement in verification_relative_url may fail if the path changes in the future. Use Django’s reverse with a named URL pattern for better maintainability.
```
verification_url = request.build_absolute_uri(
    reverse('verify-email', kwargs={'user_id': user.user_id, 'token': token})
)
```